src/main/java/util/RandomNumberGenerator.java

```diff
@@ -0,0 +1,19 @@
+package util;
+
+import java.security.SecureRandom;
+
+public class RandomNumberGenerator {
+
+    private static final int INCLUSIVE_FLAG = 1;
+    private final SecureRandom secureRandom;
+
+    public RandomNumberGenerator(SecureRandom secureRandom) {
+        this.secureRandom = secureRandom;
+    }
```

Comment:
```suggestion
    private static final SecureRandom secureRandom = new SecureRandom();
```
생성자 파라미터로 받아서 굳이 생성해줄 필요가 없을것 같아요

Reply:
해당 클래스를 생성자로 만들수 없도록 하고 (private 생성자)
메서드들을 static 메서드로해서 원하는 시점에 어디서든 사용할 수 있도록 해보면 어떨까요?
RandomNumberGenerator는 메인 로직과 큰 연관이 없고 util기능만을 하고 있는것 같은데 저는 보통 util 기능을 가진 클래스들을 이렇게 관리하곤합니다